src/components/repository-browser.js

```diff
@@ -6,34 +7,22 @@ import classNames from 'classnames';
 import prettyBytes from 'pretty-bytes';
 import selection from '../ui/selection.js';
 import Highlighter from 'react-highlighter';
+import { withUser } from './user-context.js';
 import pathHelper from '../ui/path-helper.js';
 import { inject } from './container-context.js';
 import { withTranslation } from 'react-i18next';
 import { withLanguage } from './language-context.js';
 import mimeTypeHelper from '../ui/mime-type-helper.js';
 import CdnApiClient from '../services/cdn-api-client.js';
+import { confirmCdnFileDelete } from './confirmation-dialogs.js';
+import permissions, { hasUserPermission } from '../domain/permissions.js';
 import { Input, Table, Upload, Button, message, Breadcrumb } from 'antd';
-import { translationProps, languageProps } from '../ui/default-prop-types.js';
-import { default as iconsNs, FolderOutlined, FileOutlined, CloseOutlined, UploadOutlined, HomeOutlined } from '@ant-design/icons';
+import { translationProps, languageProps, userProps } from '../ui/default-prop-types.js';
+import { default as iconsNs, FolderOutlined, FileOutlined, CloseOutlined, UploadOutlined, HomeOutlined, DeleteOutlined } from '@ant-design/icons';
 
 const Icon = iconsNs.default || iconsNs;
 
-const resourceKeyMap = {
-  [mimeTypeHelper.CATEGORY_TEXT]: 'mimeTypeCategoryText',
-  [mimeTypeHelper.CATEGORY_MARKUP]: 'mimeTypeCategoryMarkup',
-  [mimeTypeHelper.CATEGORY_IMAGE]: 'mimeTypeCategoryImage',
-  [mimeTypeHelper.CATEGORY_VIDEO]: 'mimeTypeCategoryVideo',
-  [mimeTypeHelper.CATEGORY_AUDIO]: 'mimeTypeCategoryAudio',
-  [mimeTypeHelper.CATEGORY_ARCHIVE]: 'mimeTypeCategoryArchive',
-  [mimeTypeHelper.CATEGORY_DOCUMENT]: 'mimeTypeCategoryDocument',
-  [mimeTypeHelper.CATEGORY_SPREADSHEET]: 'mimeTypeCategorySpreadsheet',
-  [mimeTypeHelper.CATEGORY_PRESENTATION]: 'mimeTypeCategoryPresentation',
-  [mimeTypeHelper.CATEGORY_PROGRAM]: 'mimeTypeCategoryProgram',
-  [mimeTypeHelper.CATEGORY_FOLDER]: 'mimeTypeCategoryFolder',
-  [mimeTypeHelper.CATEGORY_UNKNOWN]: 'mimeTypeCategoryUnknown'
-};
-
-const localizeCategory = (cat, t) => t(resourceKeyMap[cat]);
+const localizeCategory = (cat, t) => t(mimeTypeHelper.resourceKeyMap[cat]);
```

Comment:
We moved the translation keys into `mimeTypeHelper` (which is nice) but the translations are still in `repository-browser.yml`. Let's merge this but then also move those, because now they're separated.